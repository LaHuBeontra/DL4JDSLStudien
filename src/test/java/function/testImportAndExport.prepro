function main() {
	import vec3 p3, vec3 p1;

	export p1, p3;
}
