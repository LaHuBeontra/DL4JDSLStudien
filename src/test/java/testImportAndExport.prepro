function main() {
	import vec3 p1, vec3 p3;

	export p3, p1;
}
