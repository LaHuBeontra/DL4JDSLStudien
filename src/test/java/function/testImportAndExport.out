PreProDataSet{
dataSet=[42.00,  0.00,  0.00,  0.00,  0.00,  1.00,  0.00,  1.00,  0.00,  1.00,  0.00,  0.00]
[42.00,  0.00,  1.00,  0.00,  1.00,  1.00,  1.00,  1.00,  0.00,  1.00,  0.00,  1.00]
variableNames=[p1, p3]
variableShapes=[4, 3]
[4, 3]}